### DISCUSSION TITLE
PF 13.0.x Datatable glitches with BlockUI

### DISCUSSION THREAD

**Discussion by wnibes** [primefaces/community#218] [PrimeFaces] [answered]
Primefaces 13.0.1

If I use BlockUI with a DataTable, the table header appears twice when loading the page and the second header appears offset to the right below the first header; if I comment BlockUI out, everything is correct. When moving thru pages, the double header appears briefly and then disappears.

DataTable settings:
				resizeMode="expand" 
				resizableColumns="true"
				id="desktopInfoBookingsTableId"
				var="staffInfoBooking"
				value="#{viewStaffInfoBookings.lazyModel}"
				draggableColumns="true" 
				widgetVar="infoBookingsTable"
				paginator="true" 
				paginatorPosition="both"
				reflow="true"
				paginatorTemplate="{RowsPerPageDropdown} {FirstPageLink} {PreviousPageLink} {PageLinks} {NextPageLink} {LastPageLink}"
				rows="25" 
				rowsPerPageTemplate="25,50,100" 
				showGridlines="true"
				stickyHeader="true"

Could I have used something incorrectly or is this a bug?
Has anyone else observed this

**Comment by wnibes** [comment]
Answer comes by myself: The glitch appears only with sticky header set true. Without sticky header enabled, everything is fine.

**Comment by wnibes** [comment]
One more notice: When sticky header is TRUE, not only blockUI is not working fine.
Also the resizableColumns do not work (resizes only header, but not data column width)!
There is definitely a problem with that.

**Reply by melloware** [reply]
@wnibes if you create PrimeFaces Test reproducers using https://github.com/primefaces/primefaces-test and can verify the issues with PF 13.0.2 please open GitHub Issues tickets and include your reproducer.

**Reply by wnibes** [reply]
Changed to **13.0.3** and the **problem still exists**.
(the reproducer seemed quite complicated to me, which is why I haven't made any progress so far)

**Reply by melloware** [reply]
Yep need a reproducer to see the issues you are having and to make sure its not something specific to your code or project.

**Reply by wnibes** [reply]
@melloware I can understand your intention to prevent the developers from searching errrors that are based in code outside the primefaces framework.
BUT in the case described, the error can be shown by simply switching the "stickyHeader" property of DataTable on and off. I believe that it is not possible to cause the visible glitch by a backing bean... Makes no sense since it is responsive for the data not the component from PrimeFaces framework.

**Reply by melloware** [reply, marked as answer]
I just tested it with StickyHeaders 13.0.3 and its working fine for me.  Which is why I ask for a reproducer.

**Reply by melloware** [reply]
@wnibes any update? I can't reproducer your issue.
